src/tokens.ts

```diff
@@ -60,7 +60,7 @@ function getTokenTypeMap() {
   return tokens
     .split('\n')
     .map((line) => rsplit(line, '='))
-    .reduce((acum: any, [value, key]) => {
+    .reduce((acum: { [key: number]: string }, [value, key]) => {
```

Comment:
We know that the result will be an object with all `key`s being `number` and all values being `string`. This solves a type error in lines [73-76](https://github.com/solidity-parser/parser/pull/91/files#diff-b4fe2ae7aa66720c820a73560c0a9eaeef55d78e4dfc7f80efd8af7ac473fc97R73-R76)

```TypeScript
const tokenTypes = getTokenTypeMap()
...
const type = getTokenType(tokenTypes[token.type])
```